docs/attributes-registry/db.md

```diff
@@ -166,6 +166,16 @@
 | `ExecuteJavaScript` | execute_javascript |
 <!-- endsemconv -->
 
+<!-- semconv registry.db(omit_requirement_level,tag=tech-specific-cosmosdb) -->
+| Attribute  | Type | Description  | Examples  |
+|---|---|---|---|
+| [`rntbd.url`](../attributes-registry/db.md) | string | CosmosDB server url contacted | `rntbd://some-url/` |
```

Comment:
any reason not to use `url.full` ?

Also, since URL needs parsing, and since it's a transport protocol, attributes like `server.address|port` and a bunch of `network` attributes maybe more suitable.

E.g. you could do

* `network.protocol.name = rntbd` or maybe `rps.system = rntbd`
* `server.address = host-name`
* `server.port= port`
* `network.peer.address = IP contacted`
* `rntbd.status_code`
* `rntbd.sub_status_code`


PTAL at https://github.com/open-telemetry/semantic-conventions/blob/21fbd4e89462f794ac122dd44ddabee5c4a857b5/docs/rpc/rpc-spans.md to check things other protocols/rpc things do.

Reply:
Its at our hot path, I don't want to put any parsing logic there..a few extra string allocations are also a problem here.

Reply:
Rntbd is kind of close to HTTP messaging/envelope wise (Method, URI, headers, body) in that sense. 
Http also sems like following the RPC conventions, so aligning similar makes sense. 

@lmolkova Is address parsing optional or MUST? 

@FabianMeiswinkel thoughts? (might impact existing implementation)